infra/integration-image/Dockerfile

```diff
@@ -34,11 +34,23 @@ RUN curl -sL https://deb.nodesource.com/setup_7.x | bash - && \
     e2fsprogs \
     iptables \
     xfsprogs \
-    xz-utils && \
+    # Speed up ISO builds with already installed reqs
+    yum \
+    yum-utils \
+    cpio \
+    rpm \
+    ca-certificates \
+    xz-utils \
+    xorriso && \
 	# Cleanup
     apt-get autoremove -y && \
     rm -rf /var/lib/apt/lists/*
 
+COPY *.repo /etc/yum/repos.d/
+RUN yum-config-manager --enable photon photon-updates
+
+RUN mkdir -p /build/rootfs
+RUN yum --installroot /build/rootfs install coreutils filesystem linux-esx e2fsprogs gzip haveged iproute2 iptables iputils logrotate net-tools openssh photon-release procps-ng sudo systemd tdnf vim systemd -y --nogpgcheck
```

Comment:
I'm not sure I understand why we are not starting to host our own photon mirror and trying to cache packages on build time? Is bintray not providing a way to mirror? If so it shouldn't be hard to parse and download the content of https://vmware.bintray.com/photon_release_1.0_x86_64/x86_64/ via some script.

Also I don't use docker to build so I believe this pr will end up creating more confusion as I'll test a different set of versions while CI uses another set of packages.